src/gui/src/gui.tcl

```diff
@@ -170,6 +171,24 @@ proc select { args } {
     set name $keys(-name)
   }
   
+  set attribute ""
+  set value ""
+  if { [info exists keys(-filter)] } {
+    if { $name != "" } {
+      utl::error GUI 59 "-filter not available when using -name."
```

Comment:
Why not both?

Reply:
I was asking myself if it would make sense to apply this filter if we know the name of what we're trying to select. On second thought, I guess it would if we want to know if a specific object is, in this case, an IO net. Should I allow both?

Reply:
I don't see a reason not to.